data-otservbr-global/raids/raids.xml

```diff
@@ -70,6 +70,7 @@
 	<!-- Thais -->
 	<raid name="Cyclops" file="thais/cyclops.xml" interval2="500" margin="30" />
 	<raid name="OrcsThais" file="thais/orcs.xml" interval2="500" margin="70" />
+	<raid name="RatsThais" file="thais/rats.xml" interval2="500" margin="70" />
```

Comment:
duplicated: https://github.com/opentibiabr/canary/blob/main/data-otservbr-global/scripts/raids/thais/rats.lua

Reply:
Sorry, this wasn't supposed to be included in this pull.